packages/events/EventPluginHub.js

```diff
@@ -42,10 +51,10 @@ var executeDispatchesAndRelease = function(event, simulated) {
     }
   }
 };
-var executeDispatchesAndReleaseSimulated = function(e) {
+var executeDispatchesAndReleaseSimulated = function(e: ReactSyntheticEvent) {
```

Comment:
Seems like this can be inferred? I'd prefer we annotate public APIs and low-level functions, but not all the glue code.